Dockerfile

```
@@ -69,6 +62,15 @@ RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /
 
 ENV PYENV_ROOT="/pyenv-root"
 ENV PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+
+
+ARG REPO_NAME="https://github.com/itisfoundation/osparc-simcore.git"
+ARG BRANCH_NAME="master"
+ARG COMMIT_SHA="HEAD"
```

This is supposed to be pinned, `HEAD` unpins the `ooil` version coming from master. Please use the following (the last commit form master currently).
```suggestion
ARG COMMIT_SHA="638b592daed098fafbb5b846a08f6391fcb8e6b1"
```